Dispatchfile

```diff
@@ -157,7 +157,7 @@ git_resource(
     url="https://github.com/mesosphere/" + kubeaddons_kommander_git,
     revision="master")
 do_bump_kommander = bump_addon("bump-kommander", kubeaddons_kommander_git)
-do_bump_catalog="bump-kommander-catalog-api"
+do_bump_catalog="bump-catalog-api"
```

Comment:
the format "do_something" should be a function call
I don't think this variable is used except the line directly below